axelor-account/src/main/resources/views/AppInvoice.xml

```diff
@@ -9,6 +9,8 @@
             <field name="isVentilationSkipped" widget="boolean-switch"/>
 			<field name="isEnabledProductDescriptionCopyForCustomers" widget="boolean-switch"/>
 			<field name="isEnabledProductDescriptionCopyForSuppliers" widget="boolean-switch"/>
+			<field name="isEnablePDFGenerationForCustomersOnVentilation" widget="boolean-switch"/>
```

Comment:
name of these parameters should be reviewed : 
autoGenerateInvoicePrintingFileOnPurchaseInvoice
autoGenerateInvoicePrintingFileOnSaleInvoice